### ISSUE TITLE
Clarifications

### ISSUE THREAD

**Issue by gallettilance** [JamesKunstle/Adversarial-ML-GK#1]
We need to clarify in the README:
- what is the purpose of this repo?
- what is the project?
- what does a contribution look like? what should people contribute? experiments? models? datasets?

We probably also need a separate README with what you have done so far - summarizing what has been done and maybe pointing out what still needs to be done. What do you think?

**Comment by JamesKunstle** [comment]
I agree completely. I will also be adding a reference sheet and guide to relevant literature. My hope is that experiments can be proposed and discussed as projects. I'll be adding the completed experiments thus far soon as well.
